src/ops.rs

```diff
@@ -104,11 +116,17 @@ impl HttpHandler {
     }
 
     fn handle_invalid_url(&self, req: &mut Request, cause: &str) -> IronResult<Response> {
-        println!("{} requested to {} {} with invalid URL -- {}",
-                 req.remote_addr,
-                 req.method,
-                 req.url,
-                 cause.replace("<p>", "").replace("</p>", ""));
+        log!("{}{}{} requested to {}{}{} {}{}{} with invalid URL -- {}",
+             C::Green,
+             req.remote_addr,
+             CReset,
```

Comment:
Sadly there's little to be done about this, I tried grouping it and didn't arrive at an acceptable solution...